pkg/constants/indices.go

```diff
@@ -14,6 +14,11 @@ const (
 	IndexByHostName = "IndexByHostName"
 
 	IndexByClusterIP = "IndexByClusterIP"
+
+	// IndexRunningNonVclusterPodsByNode is only used when the vcluster scheduler is enabled.
+	// It maps non-vcluster pods on the node to the node name, so that the node syncer may
+	// calculate the allocatable resources on the node.
+	IndexRunningNonVclusterPodsByNode = "IndexRunningNonVclusterPodsByNode"
```

Comment:
Rename to `IndexRunningNonVClusterPodsByNode`

Reply:
Don3